packages/orchestration/test/examples/stake-bld.contract.test.ts

```diff
@@ -0,0 +1,80 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
+import { makeFakeBoard } from '@agoric/vats/tools/board-utils.js';
+import { setUpZoeForTest } from '@agoric/zoe/tools/setup-zoe.js';
+import { withAmountUtils } from '@agoric/zoe/tools/test-utils.js';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { E } from '@endo/far';
+import path from 'path';
+import { makeBankManager, makeBridge, makeLocalChain } from '../supports.js';
+
+const dirname = path.dirname(new URL(import.meta.url).pathname);
+
+const contractFile = `${dirname}/../../src/examples/stakeBld.contract.js`;
+type StartFn =
+  typeof import('@agoric/orchestration/src/examples/stakeBld.contract.js').start;
+
+test('stakeBld contract - makeAccount, deposit, withdraw', async t => {
+  const issuerKit = makeIssuerKit('BLD');
+  const stake = withAmountUtils(issuerKit);
+
+  // bootstrap
```

Comment:
```suggestion
  t.log('bootstrap')
```

consider re-ordering things slightly as in https://github.com/Agoric/agoric-sdk/blob/dc-unk-chain-misc/packages/orchestration/test/examples/unbondExample.test.ts#L99 :

```js
  t.log('bootstrap');
...
  t.log('orchestration coreEval');
...
  t.log('contract coreEval');
...
  t.log('do offer');
...
```

I prefer to make those separate functions and hence separate scopes, normally, but that turned out to be a bit of a hassle and not worth all that much here.